callstack/haul
# Added flag for disabling hot reloading.

Allows hot reloading to disabled with a flag `--hot-reloading` set to `false`.

### Summary

In our environment, hot reloading has never worked reliably and may in some cases cause crashes. We've gone so far as to have scripts to fix and strip away rules, modules, etc. from the generated configuration. Adding a flag to Haul to disable hot reloading is the right solution™.

### Test plan

- Start Haul: `yarn haul start -- --hot-reloading false`
- Make sure that the dev server still works as expected with Hot Reloading turned off
